diff -aurN quark-3.21.orig/strange-quark/quark.desktop quark-3.21/strange-quark/quark.desktop
--- quark-3.21.orig/strange-quark/quark.desktop	2003-08-30 20:59:17.000000000 +0200
+++ quark-3.21/strange-quark/quark.desktop	2004-11-24 11:51:26.095562856 +0100
@@ -1,9 +1,11 @@
 [Desktop Entry]
 Encoding=UTF-8
-Name=Quark Music Player
-Comment=Play music from the system tray
+Name=Quark
+Comment=Music player
+Comment[pl]=Odtwarzacz muzyczny
 Exec=strange-quark
 Icon=quark.png
 Terminal=false
 Type=Application
-Categories=Application;AudioVideo;
+Categories=GTK;Audio;Player;
+vi: encoding=utf-8
